code/test_odbiornik.ino

```diff
@@ -0,0 +1,57 @@
+#include <SoftwareSerial.h>
+ 
+SoftwareSerial mySerial(2, 3);
+
+unsigned long startTime = 0;
+
+
+void setup(){
+   Serial.begin(9600);
+   mySerial.begin(9600);
+   pinMode(4, INPUT);
+   while(digitalRead(4) == LOW);
+   startTime = millis();
+}
+int milisecond = 0;
+int second = 0;
+int minute = 0;
+int hour = 0;
+void denis(){
```

Comment:
formatowanie do poprawy
https://code.tutsplus.com/tutorials/top-15-best-practices-for-writing-super-readable-code--net-8118